config/locales/doc_auth/es.yml

```
@@ -253,8 +253,8 @@ es:
         trasera de su identificación.
       switch_back: Regrese a su computadora para continuar con la verificación de su
         identidad.
-      test_ssn: En el entorno de prueba solo los SSN que comienzan con “900-” se
-        consideran válidos. No ingrese PII real en este campo.
+      test_ssn: En el entorno de prueba solo los SSN que comienzan con “900-” o
+        “666-”se consideran válidos. No ingrese PII real en este campo.
```

Space after the closing quote?

```suggestion
        “666-” se consideran válidos. No ingrese PII real en este campo.
```